docs/posters-and-talks.rst

```diff
@@ -3,17 +3,47 @@ Posters and Talks
 
 This page includes various talks and posters that the Dockstore team has given.
 
+2020
+----
+
+BOSC 2020
+^^^^^^^^^
+Cabansay L, Duncan A and Yuen D. Dockstore fundamentals: Introduction to docker and descriptors for reproducible analysis [version 1; not peer reviewed]. F1000Research 2020, 9(ISCB Comm J):986 (slides) (https://doi.org/10.7490/f1000research.1118245.1)
```

Comment:
Suggest `Docker` with a capital D.

Reply:
FYI @Ldcabansay don't think I have edit access

Reply:
Tried to fix it, but I don't think you can edit on F1000 after submission unfortunately or at least I don't seem to have permissions.

Reply:
Sad